.gitignore

```diff
@@ -0,0 +1 @@
+application.properties
```

Comment:
It would be better to have `application.*` under Git, and use environment variable in it so anyone can run the application locally using some variables. This would help deployment in the future.

```suggestion
### Maven
target
!.mvn/wrapper/maven-wrapper.jar
!.mvn/wrapper/maven-wrapper.properties

```

Reply:
That's a good suggestion. I'll wait for Adrien, to find out what he has to say.

Reply:
for the moment, we really need to remove `target`. then we can look into the files coming from the IDE you are using.